cmd/nerdctl/inspect.go

```diff
@@ -148,6 +151,21 @@ func inspectAction(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func imageInspect(ctx context.Context, reqs []string, imageInspectOptions types.ImageInspectOptions) error {
+	clientOpts, err := clientutil.ProcessClientOpts(imageInspectOptions.Platform)
+	if err != nil {
+		return err
+	}
+
+	client, ctx, cancel, err := clientutil.NewClient(ctx, imageInspectOptions.GOptions.Namespace, imageInspectOptions.GOptions.Address, clientOpts...)
+	if err != nil {
+		return err
+	}
+	defer cancel()
```

Comment:
can you change this to call the func below directly (i.e., `clientutil. NewClientWithPlatform(ctx, imageInspectOptions.GOptions.Namespace, imageInspectOptions.GOptions.Address, imageInspectOptions.Platform)`

https://github.com/containerd/nerdctl/blob/67a065a80fceb7602a9092dc5dec52a959454853/pkg/clientutil/client.go#L59

Reply:
And it can be in the `inspectAction` func directly.